### ISSUE TITLE
Demo sample data

### ISSUE THREAD

**Issue by chenejac** [vivo-project/VIVO#4008]
**Is your feature request related to a problem? Please describe.**
More rich sample data are needed. The existing [sample data](https://github.com/vivo-project/sample-data/blob/main/sample-data.n3) are not sufficient for demonstrating VIVO features.   

**Describe the solution you'd like**
Synthetic data should be generated by using [Mike Conlon python script](https://github.com/mconlon17/vivo-sample-data-generator). There should be hundreds of people per institution, a dozen of institutions, and dozens of publications per person profile. Once it is generated, we should try to manually create the same data for other languages (French, Serbian, Russian, etc.)

**Additional context**
Demo sample data should be preserved into https://github.com/chenejac/sample-data/tree/reorganization/demovivo

**Comment by hauschke** [comment]
Agreed. Should the range of entities be extended, too? Grants, projects, equipment etc.
